### DISCUSSION TITLE
Для чего нужна нормализация базы данных

### DISCUSSION THREAD

**Discussion by pingvin1308** [cleannetcode/LessonMonitor#65] [MSSQL]
Для выполнения домашней работы, вам необходимо открыть и сделать форк основного репозитория по ссылке:
https://github.com/cleannetcode/LessonMonitor/tree/homework/sql%2F3

По выполнению работы, вам необходимо создать пулл реквест из вашей ветки с выполненной домашкой в основной репозиторий.

Почитать:
https://docs.microsoft.com/en-US/office/troubleshoot/access/database-normalization-description
https://habr.com/en/post/254773/
https://www.sqlservertutorial.net/sql-server-basics - Section 1-6, 11 (чтение, группировка и модификация данных)

Сделать:
![image](https://user-images.githubusercontent.com/17434920/118395593-e1193b00-b653-11eb-954f-5e5ea06ff613.png)

Проверить:
провести ревью кода любого чужого пулл реквеста относящегося к данной домашке.

**Comment by emedvedeva** [comment]
Привет! Почему то у меня никак не получается форкнуть репозиторий для этого дз. Я перехожу по ссылке в основной репозиторй (по ссылке, которая в этой дискуссии) нажимаю на fork и у меня почему-то не появляется ветка для этого дз. Что я делаю неправильно?

**Reply by eniluck** [reply]
А если добавить Remote repository? и сделать fetch из этого репозитория чтобы создать копию у себя в репозитарии... только единственное после всех этих манипуляций надо будет настроить пуш и пул запросы из твоего репозитария...

**Reply by emedvedeva** [reply]
В итоге я по новой склонировала репозиторий. Fetch  и все такое не получалось.

**Reply by eniluck** [reply]
> В итоге я по новой склонировала репозиторий. Fetch и все такое не получалось.
Не расстраивайся. Если будет время как нибудь вечером - напиши в телегу. покажу как я смог этот момент решить :)
